### PR TITLE
Extend `pl.velocity_embedding_stream` with new arguments

### PR DESCRIPTION
**Pull request by VolkerBergen**
Adding new streamplot arguments:

- arrow_size: Scaling factor for the arrow size, addressing https://github.com/theislab/scvelo/issues/486
- arrow_style: Arrow style specification, '-|>' or '->'.
- max_length: Maximum length of streamline in axes coordinates.
- integration_direction: Integration in 'forward', 'backward' or 'both' directions.

### CODE REVIEW

**Comment by VolkerBergen** [comment]
Please let me know, whether the argument names (incl. docs) make sense to you @WeilerP.

File: scvelo/plotting/velocity_embedding_stream.py [hunk 1]
```diff
@@ -201,6 +228,7 @@ def velocity_embedding_stream(
                         X_grid=None if len(vkeys) > 1 else X_grid,
                         V_grid=None if len(vkeys) > 1 else V_grid,
                         **scatter_kwargs,
+                        **stream_kwargs,
```
[the remark this answers is not in the document]
Comment (VolkerBergen) [reply scvelo/plotting/velocity_embedding_stream.py:231]: It gets updated in ln 242.

File: scvelo/plotting/velocity_embedding_stream.py [hunk 2]
```diff
@@ -161,6 +179,17 @@ def velocity_embedding_stream(
...
 
+    stream_kwargs = {
+        "linewidth": linewidth,
+        "density": density or 2,
+        "zorder": 3,
+        "arrow_color": arrow_color or "k",
+        "arrowsize": arrow_size or 1,
+        "arrowstyle": arrow_style or "-|>",
+        "maxlength": max_length or 4,
+        "integration_direction": integration_direction or "both",
```
[the remark this answers is not in the document]
Comment (VolkerBergen) [reply scvelo/plotting/velocity_embedding_stream.py:190]: If set to `None`, it gets the default value.

[the remark this answers is not in the document]
Comment (WeilerP) [reply scvelo/plotting/velocity_embedding_stream.py:190]: But `None` is not a valid option so this is redundant, right?

File: scvelo/plotting/velocity_embedding_stream.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (WeilerP) [reply scvelo/plotting/velocity_embedding_stream.py:231]: Ah yes, thanks. Missed that. Seems a bit hacky to me, though, TBH. Can it happen that we enter the if-clause multiple times?

File: scvelo/plotting/velocity_embedding_stream.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (VolkerBergen) [reply scvelo/plotting/velocity_embedding_stream.py:190]: I regularly use `None` for convenience, when I don't want to look up what the default value was.

File: scvelo/plotting/velocity_embedding_stream.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (VolkerBergen) [reply scvelo/plotting/velocity_embedding_stream.py:231]: Would use it this way for now. I don't see any danger of entering an if-clause multiple times.

**Pull request merged**